### DISCUSSION TITLE
[Probing community] Custom kernel ?

### DISCUSSION THREAD

**Discussion by SpidFightFR** [void-linux/void-packages#50059] [General]
Hey fellow Void users.

Just a quick discussion to get your thoughts about custom kernels - especially Zen or TKG.

A while back i managed to make use of zen on void, i maintained it for a few months unofficially on my side.
It worked fairly great but out of lack of interest i left the whole thing behind.

Recently i thought about doing the same for the TKG kernel, which would bring less latency, frame drops and other neat stuff like that. It's made by [TKG](https://github.com/Frogging-Family/linux-tkg) a Valve employee, whom i talked with several times in the past, including on my time on Archlinux back in the days.

He's reliable, and can be trusted.

What do you guys think ? Would you approve a potential PR ? Do you _not care_ ? Do you prefer using the default for other reasons ?

Let me know !

**Comment by classabbyamp** [comment]
we aren't going to accept packages for alternative kernels. the only non-upstream kernels in the void repos are specifically for device-specific patches, like `rpi-kernel` or `pinephone-kernel`
